scripts/Util.py

```diff
@@ -3738,24 +3738,16 @@ def getJavaArgs(self, process, current):
 
     def getSSLProps(self, process, current):
         props = Mapping.getSSLProps(self, process, current)
-        if current.config.android:
-            props.update(
-                {
-                    "IceSSL.KeystoreType": "BKS",
-                    "IceSSL.TruststoreType": "BKS",
-                    "IceSSL.Keystore": "server.bks"
-                    if isinstance(process, Server)
-                    else "client.bks",
-                }
-            )
-        else:
-            props.update(
-                {
-                    "IceSSL.Keystore": "server.jks"
-                    if isinstance(process, Server)
-                    else "client.jks",
-                }
-            )
+        props.update(
+            {
+                "IceSSL.Keystore": "server.p12" if isinstance(process, Server) else "client.p12",
+                "IceSSL.KeystorePassword": "password",
+                "IceSSL.KeystoreType": "PKCS12",
+                "IceSSL.Truststore": "ca.p12",
+                "IceSSL.TruststorePassword": "password",
+                "IceSSL.TruststoreType": "PKCS12",
+            }
```

Comment:
We now use the same configuration for both Android and Java. Additionally, the configuration has been updated to explicitly use a truststore, rather than relying on the keystore being used as the truststore when the latter is not set.